### PR TITLE
Add support for MSG_SSR_CODE_PHASE_BIASES_BOUNDS message [OTA-119]

### PR DESCRIPTION
**Pull request by fpezzinosn**
# Description

@swift-nav/devinfra

Adds the SSR_CODE_PHASE_BIASES_BOUNDS message to the SSR package.

# API compatibility

Does this change introduce a API compatibility risk?

It doesn't. It adds a new message.

## API compatibility plan

If the above is "Yes", please detail the compatibility (or migration) plan:

# JIRA Reference

https://swift-nav.atlassian.net/browse/OTA-119

### CODE REVIEW

**Comment by fpezzinosn** [comment]
@swift-nav/devinfra Any feedback on this? This should be very similar to https://github.com/swift-nav/libsbp/pull/1108 and I've incorporated here all the feedback I got there.

File: spec/yaml/swiftnav/sbp/ssr.yaml [hunk 1]
```diff
@@ -920,3 +920,57 @@ definitions:
            size_fn: n_sats
            map_by: sat_id
            desc: Orbit and Clock Bounds per Satellite
+
+ - CodeAndPhaseBiasesSatelliteSignal:
```
Comment (silverjam) [inline spec/yaml/swiftnav/sbp/ssr.yaml:924]: Any way to shorten this name?

**Review by silverjam — APPROVED** [review]
One nit, but otherwise looks good

File: spec/yaml/swiftnav/sbp/ssr.yaml (hunk 1 above)
Comment (fpezzinosn) [reply spec/yaml/swiftnav/sbp/ssr.yaml:924]: I can go with `CodePhaseBiasesSatSig`. With a previous name I hit the limit, so its a good idea to keep it short.

Comment (silverjam) [reply spec/yaml/swiftnav/sbp/ssr.yaml:924]: 👍

**Pull request merged**